Explorer/Assets/DCL/Settings/ModuleControllers/ResolutionSettingsController.cs

```diff
@@ -0,0 +1,104 @@
+﻿using DCL.Settings.ModuleViews;
```

Comment:
I think in general this controller would benefit from moving some of its functionality to common utility and/or extention methods